client/src/app/employee/employee-create/employee-create.service.ts

```diff
@@ -3,10 +3,8 @@ import { Injectable } from '@angular/core';
 import { throwError } from 'rxjs';
 import { catchError, tap } from 'rxjs/operators';
 import { EnvService } from 'src/app/shared/env.service';
-import {
-  Employee,
-  EmployeeCreate,
-} from '../employee-list/state/employee-list.model';
+import { Employee } from '../employee-detail/state/employee.model';
+import { EmployeeCreate } from '../employee-list/state/employee-list.model';
 import { EmployeeListStore } from '../employee-list/state/employee-list.store';
```

Comment:
Try to avoid sharing your store beyond the one service it is associated with. If you start passing around the store to other consumers it becomes harder to predict where state changes are coming from.

Instead, this behavior should probably be in the `employee-list.service` and that service should be used in the `employee-create.component`.

Reply:
removed `employeeCreateService`, added method to `employeeListService`